src/containers/App/App.js

```diff
@@ -10,7 +10,7 @@ export const App = () => {
                 {routesConfig.map((route) => (
                     <Route key={route.path} path={route.path} {...route} />
                 ))}
-                <Redirect to={"/404"} />
+                <Redirect to={"/nb/"} />
```

Comment:
Denne må endres til <Redirect to="/:lng/404"> slik at man blir redirectet til 404 siden (med riktig språk) dersom man prøver å gå til en URL som ikke er definert i routeConfig.

Men vi må også håndtere dersom man forsøker å nå løsningen uten språkkode, så må denne legges på.
Dvs. "/pensjon/opptjening" => "/pensjon/opptjening/nb"
Usikker på hvor og hvordan denne endringen må gjøres.....spør google (??)